portfolio/src/main/webapp/script.js

```diff
@@ -35,11 +35,65 @@ function addRandomFact() {
     
     //remove this fact and the null from the array
     facts.splice(factIndex,1);
-    console.log(facts.toString());
-    console.log(facts.length);
 
     //update i 
     i = facts.length-1;
   }
+}
+
+
 
+// function to calculate the result of the survey
+function tabulateAnswers() {
+  // initialize variables for each choice's score
+  // If you add more choices and outcomes, you must add another variable here.
+  var c1score = 0;
+  var c2score = 0;
+  var c3score = 0;
+  var c4score = 0;
+  
+  // get a list of the radio inputs on the page
+  var choices = document.getElementsByTagName('input');
```

Comment:
This would grab all inputs on the page.  It works ok now since you only have the quiz on there.  But when you add more input elements (eg. if you add a search bar or a subscribe email box) then this would grab those as well.

One option would be to limit the scope of the getter:
`var choices = document.getElementById('quiz').getElementsByTagName('input');`

You can also add a class to all of your quiz related inputs then do:
`var choices = document.getElementsByClassName('quiz-inputs');`